src/components/Login/Login.js

```diff
@@ -34,6 +33,8 @@ class Login extends Component {
         this.setState({ loaded: true });
       } else if (result) {
         UserUtil.getMyProfile(this.onTokenValidCheck.bind(this));
+      } else {
+        this.setState({ loaded: true });
```

Comment:
Trevor. Look! AsyncStorage is React Native component.
It is a component problem.